### DISCUSSION TITLE
Displaying the percent sign (%) at the end of the right prompt

### DISCUSSION THREAD

**Discussion by bound-variable** [reobin/typewritten#153] [Q&A] [answered]
In my .zshenv, I've got:
```
battery() {
  local capacity=$(cat /sys/class/power_supply/BAT0/capacity)
  echo "${capacity}%"
}

export TYPEWRITTEN_RIGHT_PROMPT_PREFIX_FUNCTION=battery
```
But my right prompt displays a tilde (~) instead of a percent sign (%).
![image](https://user-images.githubusercontent.com/51193876/199627150-a9ee4133-05d6-4015-af36-0544b7eae3ab.png)

I've also tried with printf.

Any ideas how to get the percent sign to display?

**Comment by reobin** [comment, marked as answer]
Hey @Pound-Hash !

That is actually a great use of the prefix function!

Try doubling your percentage sign:

```sh
battery() {
  local capacity=$(cat /sys/class/power_supply/BAT0/capacity)
  echo "${capacity}%%"
}
```

**Reply by bound-variable** [reply]
Thanks. I actually tried that and noticed the issue. The tilde was on the right because that was the indication for my home directory. So that makes sense now. However, typewritten doesn't seem to have the ability to format the prompt in the way I want. I'd prefer to have the path information (with tilde) and git information above the prompt on the left, and have my battery indication on the right, by itself. If typewritten is updated in the future with this feature, I'll stick to it.

I wanted to delete this question because I had already noticed the issue. But thanks for your reply!
